### ISSUE TITLE
Persistent fever

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6171]
**Preferred term label:**
Persistent fever

**Synonyms**
Lasting fever

**Definition (free text, please give PubMed ID)**
Fever lasting longer than usual.
DOI: https://doi.org/10.1080/10428194.2019.1691198

**Parent term (use hpo.jax.org/app)**
Fever

**Comment by pnrobinson** [comment]
Adding term as HP:0033399 with
def: Fever that persists longer than expected for an acute infectious disease. No precise threshold exists, and the clinical interpretation of fever is context-dependent, but as a rule of thumb, this term refers to fever that persists longer than 2-3 weeks.
comment:In most patients with fever lasting 1 to 2 weeks, the etiology will be found or the fever will disappear. If fever persists beyond this point, a patient is said to have a said to have fever of undetermined origin. The distinction between 'fever of unknown origin' (FUO) and 'Persistent fever' is that FUO refers to a febrile disease whose cause is yet unknown, and persistent fever is a clinical manifestation of an FUO or of a disease of known etiology that is characterized by fever that last longer than 2-3 weeks.
